.github/workflows/e2e.yml

```diff
@@ -67,7 +71,26 @@ jobs:
           # https://github.com/cypress-io/github-action/issues/124#issuecomment-653180260
           COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
 
-          # Visual Testing config
-          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
-          PERCY_PARALLEL_TOTAL: 3 # remember to keep this synced with matrix.containers
-          PERCY_BRANCH: ${{ github.head_ref }}
+  finalize-happo:
+    name: Finalize Happo
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
```

Comment:
What does this mean: `if: ${{ always() }}`?

It looks a bit odd, if we always want to run it, couldn't we leave the `if` statement out?

Reply:
TL;DR This is the way GitHub Actions allows you to run a postscript regardless of the previous step result.

Each job in the CI will be run only if the previous job is finished successfully. Some CI tools have a specific property for this, like [`after_script` from Travis CI](https://blog.travis-ci.com/after_script_behavior_changes/).

GitHub Actions doesn't have a specific property to do this, so you need to indicate which is the dependent step with `needs` + indicate to run it no matter what's the result from the previous step with `if: always` to get the same behavior.

We need this behavior to tell Happo to finalize the screenshot process after Cypress tests, no matter what's the result from Cypress itself.

Reply:
> We need this behavior to tell Happo to finalize the screenshot process after Cypress tests, no matter what's the result from Cypress itself.

Ah that's the key bit that I didn't understand from reading the workflow file. 😄